.gitignore

```diff
@@ -98,6 +98,7 @@ flamegraph.pl
 
 # Celery
 celerybeat-schedule
+celerybeat-schedule.db
```

Comment:
My celery creates a new schedule file, so I think it should be added to .gitignore